docs/content/faq.md

```diff
@@ -68,3 +68,15 @@ the meta tags using the standard authentication mechanisms supported by `go`.
 Therefore, if `go` before v1.11 worked for you, then go 1.11+ with GOPROXY
 should work as well, provided that the Athens proxy host is configured with the 
 proper authentication.
+
+### Is there support for monitoring and observability for Proxy?
+
+Right now, we have structured logs for proxy. Along with that, we have added tracing to help developers identify critical code paths and debug latency issues. While there is no setup required for logs, tracing requires some installation. We currently store our trace exports using [Jaeger](https://www.jaegertracing.io/). Further support for other exporters is in progress.  To try out tracing, do the following:
+
+- Set the environment to development (otherwise traces will be sampled)
+- Run `docker-compose up -d` from `cmd/proxy`
+- Run the walkthrough tutorial
+- Open `http://localhost:16686/search`
+
+Observability is not a hard requirement for the proxy. So, if the infrastructure is not properly set up, they will fail with an information log.
```

Comment:
can you explain a little more about what it means for the infrastructure to not be properly set up? for example, does it mean that you didn't run `docker-compose up` from `cmd/proxy`?

Reply:
My bad, I was writing of my memory last night. It would just be `docker-compose up -d` from the root directory. I was thinking that I was in `cmd/proxy`

Reply:
```
Observability is not a hard requirement for the proxy. So, if the infrastructure is not properly set up, they will fail with an information log. For eg. if Jaeger is not running / if the wrong URL to the exporter is provided, proxy will continue to run. However, it will not collect any traces or metrics(when it is available).
```
Does this sound good?